site/realm/functions/logIncidentHistory.js

```diff
@@ -0,0 +1,31 @@
+
+/**
+ * This function creates a entry into the Incident History collection
+ * 
+ * @param {object} input A JSON object that contains the Incident data
+ * 
+*/
+exports = async (input) => {
+
+  const incidentsHistory = context.services.get('mongodb-atlas').db('history').collection("incidents");
+
+  const {
+    AllegedDeployerOfAISystem,
+    AllegedDeveloperOfAISystem,
+    AllegedHarmedOrNearlyHarmedParties,
+    ...incident } = input;
+
+  incident['Alleged deployer of AI system'] = AllegedDeployerOfAISystem;
```

Comment:
Why did you need to remove spaces here?

Reply:
Actually, we need to add the spaces in order to insert it into the DB

Reply:
Oh I remember, the auto-generated graphql endpoint removes spaces and uses CamelCase